src/native/managed/cdac/mscordaccore_universal/Legacy/ClrDataAddress.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices.Marshalling;
+
+namespace Microsoft.Diagnostics.DataContractReader.Legacy;
+
+/// <summary>
+/// Represents the native CLRDATA_ADDRESS 64-bit type which uses sign extending
+/// when converting from 32-bit values to 64-bit values.
+/// For more information see TO_CDADDR in dacimpl.h
```

Comment:
Nit: We hope dacimpl.h is going to disappear eventually, so this comment is not durable